src/java.management/share/classes/com/sun/jmx/remote/security/MBeanServerAccessController.java

```diff
@@ -174,15 +168,8 @@ public ObjectInstance createMBean(String className, ObjectName name)
         MBeanException,
         NotCompliantMBeanException {
         checkCreate(className);
-        @SuppressWarnings("removal")
-        SecurityManager sm = System.getSecurityManager();
-        if (sm == null) {
-            Object object = getMBeanServer().instantiate(className);
-            checkClassLoader(object);
```

Comment:
I believe we should keep this call to `checkClassLoader(object)` in place for now. What to do in this case will require some more thinking - I'd suggest handling that part in a separate PR.

Reply:
Yes agreed.